index.js

```diff
@@ -0,0 +1,27 @@
+const hamburger = document.querySelector('.menu-bars');
+const nav = document.querySelector('#desktop-nav');
+const menulist = document.querySelector('#desktop-nav ul');
+const menuitems = Array.from(document.querySelector('#desktop-nav ul li'));
+const icon = document.querySelector('.menu-bars .fas');
+
+hamburger.addEventListener('click', () => {
+  nav.classList.toggle('nav-open');
+  menulist.classList.toggle('nav-open');
+
+  if (nav.classList.contains('nav-open')) {
+    icon.classList.remove('fa-bars');
+    icon.classList.add('fa-times');
+  } else {
+    icon.classList.remove('fa-times');
+    icon.classList.add('fa-bars');
+  }
+});
+
+menuitems.forEach((item) => {
+  item.addEventListener('click', () => {
+    nav.classList.toggle('nav-open');
+    menulist.classList.toggle('nav-open');
+    icon.classList.remove('fa-times');
+    icon.classList.add('fa-bars');
+  });
```

Comment:
- [ ] Great job Implementing the `menuitems` loop to close the navigation mobile menu page 👍🏻  however it does not work yet, I'm still seing the menu option on the top when I click any option of the mobile menu, but as required when the user clicks (or taps) any option of the mobile menu, the mobile menu must disappear. Kindly fix this